mmdet/models/dense_heads/fcos_head.py

```diff
@@ -219,6 +219,7 @@ def loss(self,
         pos_inds = ((flatten_labels >= 0)
                     & (flatten_labels < bg_class_ind)).nonzero().reshape(-1)
         num_pos = len(pos_inds)
+        num_pos = max(reduce_mean(torch.tensor(num_pos).cuda()).item(), 1.0)
```

Comment:
Casting like the below may be needed for PyTorch 1.6+. https://github.com/open-mmlab/mmdetection/pull/4210
``` python
num_pos = torch.tensor(num_pos, dtype=torch.float, device=bbox_preds[0].device)
num_pos = max(reduce_mean(num_pos).item(), 1.0)
```